ADAL/src/ADAuthenticationErrorMap.m

```diff
@@ -0,0 +1,130 @@
+                                   @(MSIDErrorUnsupportedFunctionality): @(AD_ERROR_UNEXPECTED),
+                                   // Cache
+                                   @(MSIDErrorCacheMultipleUsers) : @(AD_ERROR_CACHE_MULTIPLE_USERS),
+                                   @(MSIDErrorCacheBadFormat) : @(AD_ERROR_CACHE_BAD_FORMAT),
+                                   // Authority Validation
+                                   @(MSIDErrorAuthorityValidation) : @(AD_ERROR_DEVELOPER_AUTHORITY_VALIDATION),
+                                   // Interactive flow
+                                   @(MSIDErrorUserCancel) : @(AD_ERROR_UI_USER_CANCEL),
+                                   @(MSIDErrorSessionCanceledProgrammatically) : @(AD_ERROR_UI_USER_CANCEL),
+                                   @(MSIDErrorInteractiveSessionStartFailure) : @(AD_ERROR_UNEXPECTED),
+                                   @(MSIDErrorInteractiveSessionAlreadyRunning) : @(AD_ERROR_UI_MULTLIPLE_INTERACTIVE_REQUESTS),
+                                   @(MSIDErrorNoMainViewController) : @(AD_ERROR_UI_NO_MAIN_VIEW_CONTROLLER)
+                                   },
+                           ADOAuthServerErrorDomain:@{
+                                   @(MSIDErrorInteractionRequired) : @(AD_ERROR_SERVER_USER_INPUT_NEEDED),
+                                   @(MSIDErrorServerOauth) : @(AD_ERROR_SERVER_OAUTH),
+                                   @(MSIDErrorServerInvalidResponse) : @(AD_ERROR_SERVER_INVALID_RESPONSE),
+                                   @(MSIDErrorServerRefreshTokenRejected) : @(AD_ERROR_SERVER_REFRESH_TOKEN_REJECTED),
+                                   @(MSIDErrorServerInvalidRequest) :@(AD_ERROR_SERVER_OAUTH),
+                                   @(MSIDErrorServerInvalidClient) : @(AD_ERROR_SERVER_OAUTH),
+                                   @(MSIDErrorServerInvalidGrant) : @(AD_ERROR_SERVER_OAUTH),
+                                   @(MSIDErrorServerInvalidScope) : @(AD_ERROR_SERVER_OAUTH),
+                                   @(MSIDErrorServerInvalidState) : @(AD_ERROR_SERVER_OAUTH),
+                                   @(MSIDErrorServerNonHttpsRedirect) : @(AD_ERROR_SERVER_NON_HTTPS_REDIRECT),
+                                   @(MSIDErrorServerProtectionPoliciesRequired) : @(AD_ERROR_SERVER_PROTECTION_POLICY_REQUIRED),
+                                   @(MSIDErrorAuthorizationFailed): @(AD_ERROR_SERVER_AUTHORIZATION_CODE)
+                                   },
+                           ADHTTPErrorCodeDomain: @{
+                                   @(MSIDErrorServerUnhandledResponse) : @(AD_ERROR_UNEXPECTED)
+                                   }
+                           };
+    
+    s_userInfoKeyMapping = @{
+                             MSIDHTTPHeadersKey : ADHTTPHeadersKey,
+                             MSIDOAuthSubErrorKey : ADSuberrorKey
```

Comment:
There was one more key added in dev, make sure you apply latest changes (I think it was user displayable id)